spring-core/src/main/java/org/springframework/util/comparator/Comparators.java

```diff
@@ -29,49 +29,44 @@
 
 	/**
 	 * Return a {@link Comparable} adapter.
```

Comment:
I've used `Comparator::naturalOrder` and `Comparator::nullLast/nullFirst` instead of `NullSafeComparator`